firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/analytics/AnalyticsConnectorReceiver.java

```diff
@@ -91,6 +102,25 @@ public void onMessageTriggered(int id, @Nullable Bundle extras) {
       params = new Bundle();
     }
 
+    final String origin = params.getString(EVENT_ORIGIN_KEY);
+    if (CRASHLYTICS_ORIGIN.equals(origin)) {
+      fireCrashlyticsOriginEvent(id, extras);
```

Comment:
nit: the "fire" prefix is... slightly overloaded in this codebase.

Reply:
🔥🔥🔥 Whatever do you mean? 🔥🔥🔥